src/SimpleCLIUserInterface.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) Sven Rosvall (sven@rosvall.ie)
```

Comment:
Please put your name in these copyright notices.

Reply:
I have started to do that in the example files. I think we should agree who should be in what sections. If (as I now agree) this is to be a separate repository I want to agree a name for the repository as well.